irohad/torii/processor/transaction_processor_impl.hpp

```diff
@@ -19,6 +19,9 @@
 #define IROHA_TRANSACTION_PROCESSOR_STUB_HPP
 
 #include <mutex>
+
+#include <rxcpp/rx.hpp>
```

Comment:
Please do not include whole rx. In this file only rx-subjects are needed.

Reply:
Decided that it's close to impossible not to include the whole rx here